tests/Doctrine/Tests/Models/CMS/CmsAddress.php

```diff
@@ -21,74 +21,25 @@
 use Doctrine\ORM\Mapping\SqlResultSetMappings;
 use Doctrine\ORM\Mapping\Table;
 
-/**
- * CmsAddress
- *
- * @Entity
- * @Table(name="cms_addresses")
- * @SqlResultSetMappings({
```

Comment:
Todo: figure out what's up with these annotations: why are they removed, why are other similar annotations in other files not removed, and why removing them does not cause any test to fail… maybe the corresponding feature was removed?

Reply:
Yes it was removed

Reply:
Ah thanks! I'll remove these annotations from lower branches then.

Reply:
:thinking: I think it was removed in https://github.com/doctrine/orm/pull/7095, but that PR was not backported, and the feature is not deprecated in 2.x

Reply:
That's weird because it is featured in https://github.com/doctrine/orm/projects/4#card-63894409 , and marked as done. I'll finish the job.

Reply:
See https://github.com/doctrine/orm/pull/10114